FireboltDotNetSdk.Tests/Integration/ExecuteTest.cs

```diff
@@ -52,17 +53,28 @@ public void ExecuteSelectTest(string commandText)
             executeTest(SYSTEM_CONNECTION_STRING, commandText, null, true);
         }
 
-        [Test, Timeout(600000)]
+        [Test, Timeout(1200000)]
         [Category("slow")]
         public void ExecuteLongTest()
         {
+            var watch = System.Diagnostics.Stopwatch.StartNew();
             using var conn = new FireboltConnection(SYSTEM_CONNECTION_STRING);
             conn.Open();
             DbCommand command = conn.CreateCommand();
             command.CommandTimeout = 0; // make command timeout unlimited
-            command.CommandText = "SELECT checksum(*) FROM generate_series(1, 200000000000)";
+            // Use more rows for FB2.0 since it seems to be faster
+            var max = UserName == null ? 400000000000 : 250000000000;
```

Comment:
Wouldn't that overcomplicate this test?
We would still have this ugly logic and we'll have another abstraction function.
Maybe you have a better vision for it, in such case can you share the example of how you think this should be done

Reply:
IMHO not because this adds some kind of granularity and visibility, also this helps to add support to v3. However it is not the thing I would fight for.